ckanext/ontario_theme/templates/internal/package/new_resource_publish.html

```diff
@@ -1,16 +1,20 @@
-{% extends "package/base.html" %}
+{% extends "package/resource_edit_base.html" %}
```

Comment:
Can you revert back to `package/base.html`? This will be necessary for the actual content of this step.

Reply:
Since `resource_edit_base.html` extends `base.html` this change won't break anything, but it will keep the page consistent with all the resource edit pages (i.e. the pages that use the step indicator and any pages from the tabs in the edit resource page)

Reply:
I changed the name from `res` to `resource` in the `plugin.py` (and set res to equal resource in the template to avoid changing all instances of res) and it was fine.